### ISSUE TITLE
 ruleset_converter: built to be run on Android

### ISSUE THREAD

**Issue by ac130kz** [bromite/bromite#1770]
### Bromite version

Version: `96.0.4664.183` 
Arch: `arm64` 
Android version: `11.0` 
Device model: `Poco F1 (beryllium)` 

### Flags

These are the flags changed under `chrome://flags`:

### Is this bug about the SystemWebView?

No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No

### Is this bug happening in an incognito tab?
No

### Allow ads for the site; is the bug caused by the adblocker?
No

### Is the bug a crash?

No

### Describe the bug

I'm unable to run `ruleset_converter` on my x86_64 machine.

### Steps to reproduce the bug

```
$ file ruleset_converter
ruleset_converter: ELF 32-bit LSB pie executable, ARM, EABI5 version 1 (SYSV), dynamically linked, interpreter /system/bin/linker, BuildID[sha1]=d3175623144129e76bad3531a826a2917706ced1, stripped
```

### Expected behavior

The converter should be a x86_64 (or maybe an ARM) executable, which runs without Android.

**Comment by kokukyocho** [comment]
Hello @ac130kz!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md); the issue will be automatically re-opened afterwards.

If this is not an issue nor a feature request and you want to start a discussion then you can use [GitHub Discussions](https://github.com/bromite/bromite/discussions).

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.

**Comment by csagan5** [comment]
Fixed in `97.0.4692.106`.
